integration_test/cases/migrator.exs

```diff
@@ -81,6 +86,9 @@ defmodule Ecto.Integration.MigratorTest do
 
       assert count_entries() == 1
       assert [50] = run(TestRepo, path, :up, to: 50, log: false)
+
+      purge migrations
```

Comment:
Ah, yes, you need to purge if you run them again. Good point. The point though is to remove those changes altogether and just make sure we clean up the table before running the tests. Can you update this PR?